README.md

```diff
@@ -1,27 +1,140 @@
-# Docs Scraper
+# Docs Scraper <!-- omit in TOC -->
 
-A scraper for MeiliSearch's documentation, indexing the content into a MeiliSearch instance.
+A scraper for your documentation website, indexing the content into a MeiliSearch instance.
```

Comment:
This sentence is confusing. This is a scraper that will scrap your documentation and index the content into a MeiliSearch instance? If this is the case : 
```suggestion
A scraper for your documentation website that indexes the scraped content into a MeiliSearch instance.
```